libpod/runtime_img.go

```diff
@@ -268,12 +271,27 @@ func (r *Runtime) LoadImage(ctx context.Context, name, inputFile string, writer
 			newImages, err = r.ImageRuntime().LoadFromArchiveReference(ctx, src, signaturePolicy, writer)
 		}
 		if err != nil {
-			src, err := directory.NewReference(inputFile)
-			if err == nil {
-				newImages, err = r.ImageRuntime().LoadFromArchiveReference(ctx, src, signaturePolicy, writer)
+			if strings.Contains(name, "/") {
+				// prepend "localhost/" to support local image saved with this semantics
+				src, err = ociarchive.NewReference(inputFile, filepath.Join("localhost", name))
```

Comment:
Not `filepath.Join`, that is semantically incorrect and it would use backslashes on Windows. Even using `"localhost/" + ` would be better.

But there already is some code around for dealing with this, at least this should use `DefaultLocalRegistry`.